oaknational/Oak-Web-Application
# feat: search filters from oak components - LESQ-766

## Description

Music year: {owa_music_year}

- uses Oak Components search filter checkbox
- Updated designs for search page 
- Move Unit and Lesson below search bar
- Refactor search events for new component 

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2400--oak-web-application.netlify.thenational.academy
Go to search page - checkbox's should be new Oak Search Checkbox
Events searchRefined should be sent to avo / posthog

## Checklist
